core/chains/evm/config/toml/defaults/Hedera_Mainnet.toml

```diff
@@ -0,0 +1,33 @@
+ChainID = '295'
+ChainType = 'hedera'
+#considering the 3-5 (6 including a buffer) seconds of finality and 2 seconds block production
+#we set the depth to 6/2 = 3 blocks, setting to 10 for safety
+FinalityDepth = 10
+#Hedera has high TPS, so polling less often 
+LogPollInterval = '10s'
```

Comment:
If block rate is 2 seconds, isn't 10 seconds too slow?

Reply:
Hedera nodes lags in syncing the transactions from the mirror node & the txns are not available within 2 seconds, hence the gap
Also afair this doesnt have any impact on the Data feeds, but I think I still saw some warnings/issues due to this

Reply:
I guess LogPollInterval will only affect delay until log is received by the application layer. So I'm ok with `10s`. 
cc @reductionista to confirm